sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -111,7 +111,7 @@ def begin_recognize_receipts(self, receipt, **kwargs):
             and/or ranges of pages you want to get in the result. For a range of pages, use a hyphen, like
             `pages=["1-3", "5-6"]`. Separate each page number or range with a comma.
         :return: An instance of an LROPoller. Call `result()` on the poller
-            object to return a list[:class:`~azure.ai.formrecognizer.RecognizedForm`].
+            object to return a list[:class:`~azure.ai.formrecognizer.RecognizedForm`]].
```

Comment:
doesn't look like we need the extra bracket here or am I missing something?

edit: it seems like only the rtype needs correction?

Reply:
Oops! A find error. Fixing

Reply:
Done